x/interchainstaking/keeper/keeper.go

```diff
@@ -347,6 +353,14 @@ func (k *Keeper) SetValidatorForZone(ctx sdk.Context, zone *types.Zone, data []b
 		}
 	}
 
+	// checking whether validator was tombstoned or not
+	if val.Jailed {
+		err := k.EmitSigningInfoQuery(ctx, zone.ConnectionId, zone.ChainId, validator)
```

Comment:
Store validator.ConsensusAddr against val (need to add new field).

Reply:
Instead of adding a new field, add a new mapping between consaddress and valoper address.

We only need to store this for vals at the point we trigger a tombstone lookup.